src/Execution/DataLoader/RelationAvgLoader.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Nuwave\Lighthouse\Execution\DataLoader;
+
+use Closure;
+use Illuminate\Database\Eloquent\Collection as EloquentCollection;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Str;
+
+class RelationAvgLoader implements RelationAggregateLoader
+{
+    /**
+     * @var \Closure
+     */
+    protected $decorateBuilder;
+
+    public function __construct(Closure $decorateBuilder)
+    {
+        $this->decorateBuilder = $decorateBuilder;
+    }
+
+    public function load(EloquentCollection $parents, string $relationName, string $column): void
+    {
+        self::loadAvg($parents, [$relationName => $this->decorateBuilder], $column);
```

Comment:
Why not use `EloquentCollection::loadAvg()`? Same with the other aggregate loaders.